src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,15 @@
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User | null;
+}
 // Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo: React.FC<Props> = ({ user }) => (
```

Comment:
use destructurization
```suggestion
export const UserInfo: React.FC<Props> = ({ user: { ... } }) => (
```